lib/alephant/broker/component.rb

```diff
@@ -35,7 +35,9 @@ def headers
       end
 
       def status
-        data[:meta].key?("status") ? data[:meta]["status"] : 200
+        meta = data.fetch(:meta, {})
+
+        meta[:status] || meta['status'] || 200
```

Comment:
Not yet 😢 

All of the old strategy code etc uses a mix (via Hashie::Mash).  Once we've updated that we can kill most of the duplication.